tealium/collectors/media/MediaContent.swift

```diff
@@ -29,6 +29,9 @@ public class MediaCollection: Codable {
     var adBreaks = [AdBreak]()
     var ads = [Ad]()
     var chapters = [Chapter]()
+    static var numberOfAds = 0
```

Comment:
Perhaps computed properties that count the number of items in `adBreaks`? Same comment for `numberOfAdBreaks` and `numberOfChapters`.